about.html

```diff
@@ -84,9 +87,7 @@ <h2 class="mb-0">
     </div>
   </div>
     </div>
-    <div class="col-sm-4">
```

Comment:
I moved the image to the top, but I didn't really have to. The two different `order` classes were enough to fix the problem but I didn't know that at first.